plugins/service_discovery/srv.md

```diff
@@ -0,0 +1,105 @@
+# SRV Service Discovery Plugin
+
+The `srv` serivce discovery plugin updates targets by [SRV Record](https://tools.ietf.org/html/rfc2782)
+
+## Example Configuration
+
+This is an exmple with out\_forward.
+It udpates targets to get SRV record from `_fluentd._tcp.exmaple.com`.
+
+```
+<source>
+  @type forward
+
+  <service_discovery>
+    @type srv
+    service fluentd
+    proto tcp
+    hostname example.com
+  </service_discovery>
+</source>
+```
+
+## Parameters
+
+### @type
+
+The value must be `srv`.
+
+### service
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Service without underscore in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+### proto
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Proto without underscore in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+### hostname
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Name in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+
+### dns\_server\_host
+
+| type   | default | version |
+|:-------|:------------------------|:--------|
```

Comment:
column style glitch? But this comment is nitpick because GitBook renders no glitch table.

Reply:
fixed https://github.com/fluent/fluentd-docs-gitbook/pull/160/commits/cf936ef481bf5fb685fb783d92efca30d098d50f